### ISSUE TITLE
Fail should not suppress output.

### ISSUE THREAD

**Issue by gares** [coq-community/vscoq#819]
```coq
Check nat nat. (* error message displayed *)
Fail Check nat nat. (* no message displayed, while we are used to see the error message turned into an info one *)
```

**Comment by gares** [comment]
https://github.com/coq/coq/blob/c0c8be2c4f575034ff23c7972393cdc324df2334/vernac/vernacinterp.ml#L41-L59

**Comment by rtetley** [comment]
Closed by #842
